internal/commands/pull/command_test.go

```diff
@@ -441,14 +440,21 @@ Successfully pulled app down: app
 			profile, teardown := mock.NewProfileFromTmpDir(t, "profile_name")
 			defer teardown()
 
-			_, ui := mock.NewUI()
+			_, console, _, ui, err := mock.NewVT10XConsole()
+			assert.Nil(t, err)
+			defer console.Close()
+			go func() {
+				console.ExpectString("Would you like to export with a client template?")
```

Comment:
is this still needed?  if so, we need to make sure we wait on it either through ` doneCh` or `waitgroup`, but I think we just revert this and we're ok right?